openbb_platform/core/openbb_core/provider/standard_models/sp500_multiples.py

```diff
@@ -12,7 +12,7 @@
     QUERY_DESCRIPTIONS,
 )
 
-SERIES_NAMES = Literal[
+SeriesNames = Literal[
```

Comment:
Either keep this `SERIES_NAME` or do it all lowercaps to stick with a pythonic way.